### PR TITLE
Unit testing for logging module added

### PR DESCRIPTION
**Pull request by kannankvs**
##### SUMMARY
Unit testing for logging module added

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
logging Unit Testing

### CODE REVIEW

File: tests/unit/modules/network/sonic/sonic_module.py
Comment (kerry-meyer) [inline tests/unit/modules/network/sonic/sonic_module.py:1]: Why does this function become necessary with the addition of the "logging" UT? (What does it do? Why wasn't it needed for UT in other resource modules?

File: tests/unit/modules/network/sonic/test_sonic_logging.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    def setUp(self):
+        super(TestSonicInterfacesModule, self).setUp()
+        self.facts_edit_config = self.mock_facts_edit_config.start()
+        self.config_edit_config = self.mock_config_edit_config.start()
+        self.facts_edit_config.side_effect = self.facts_side_effect
+        self.config_edit_config.side_effect = self.config_side_effect
+        self.get_interface_naming_mode = self.mock_get_interface_naming_mode.start()
+        self.get_interface_naming_mode.return_value = 'standard'
+        self.utils_send_requests = self.mock_utils_send_requests.start()
+        self.utils_send_requests.side_effect = self.send_requests_side_effect
```
Comment (kerry-meyer) [inline tests/unit/modules/network/sonic/test_sonic_logging.py:45]: Related to the question asked on the sonic_module file: What is the purpose of this additional setup step?

**Review by kerry-meyer — COMMENTED** [review]
I have posted a couple of questions about the addition of the "send_requests_side_effect" function in the UT for this resource module. I am not expecting these to result in change requests, but would like to understand the functionality before approving and merging this PR.

**Comment by kannankvs** [comment]
@kerry-meyer : Modified the mocking method for send_requests; reused config_side_effect instead of introducing new mocking method. Previously, I was thinking that mocking send_requests differently might be required in many modules. Its now removed and same old config_side_effect is reused.

File: tests/unit/modules/network/sonic/sonic_module.py
Comment (kerry-meyer) [reply tests/unit/modules/network/sonic/sonic_module.py:1]: This file has been deleted from the change set.

I am marking this issue "Resolved".

File: tests/unit/modules/network/sonic/test_sonic_logging.py (hunk 1 above)
Comment (kerry-meyer) [reply tests/unit/modules/network/sonic/test_sonic_logging.py]: This code has been revised.

I am marking this issue "Resolved".

**Pull request merged**